pipeline/config-dev.groovy

```diff
@@ -235,28 +235,28 @@ environments {
             }
             resources {
                 node {
-                    cpu_request = "50m"
-                    cpu_limit = "100m"
-                    memory_request = "256Mi"
-                    memory_limit = "384Mi"
-                    replica_min = 1
-                    replica_max = 1
+                    cpu_request = "100m"
```

Comment:
Was this fine tuning done as part of this PR, or is this in conflict with changes that @FreshGyan  was doing earlier this week?

Reply:
This was done for Brian's demo, I think the last of which is today. I'll revert the changes after.